### PR TITLE
Docs: Part 7

### PR DESCRIPTION
**Pull request by junior1q94**
Closes [Ticket](https://trello.com)

The Seach App. Consuming the Fetch API.

## Testing

Steps to manually verify the change:
1. _Setup_: cd docs/
2. _Run_: make html 
3. _Verify_: You can check the output of the above command in the _build/ directory

### CODE REVIEW

File: docs/tutorial/part07.rst [hunk 1]
```diff
@@ -1,6 +1,117 @@
...
-How to connect your frontend with the API.
-==========================================
+========================================
+The Search App, consuming the Fetch API.
+========================================
 
-.. todo::
-   Here will be the tutorial body.
\ No newline at end of file
+At this point, the API is ready. Congratulation!
```
Comment (mariamera) [inline docs/tutorial/part07.rst:5]: Congratulations*

File: docs/tutorial/part07.rst [hunk 2]
```diff
@@ -1,6 +1,117 @@
...
+    .fail(() => {
+        // Handle errors
+        console.log("Fetch error");
+    })
+    .always(() => {
+        // Always do something at end
+    });
+
+
+Currently the Search App is not making any requests. Instead it is using some mockup data. But
```
Comment (mariamera) [inline docs/tutorial/part07.rst]: Instead, it is using some mockup data

File: docs/tutorial/part07.rst [hunk 3]
```diff
@@ -1,6 +1,117 @@
...
+
+.. figure:: _images/part07-img3.png
+   :width: 100%
+   :alt: Node Details
+
+   That is modal with more details of a specific node. We will see, for example,
+   the connections of Entity node with others Officer or Intermediary nodes.
+
+The tools used to build the Paradise Paper Search App frontend are probably pretty familiar to
+you(JavaScript, KnockoutJS, JQuery, HTML, Bootstrap). We will not give much details about its
```
Comment (mariamera) [inline docs/tutorial/part07.rst]: We will not give many details

File: docs/tutorial/part07.rst [hunk 4]
```diff
@@ -1,6 +1,117 @@
...
+    })
+    .always(() => {
+        // Always do something at end
+    });
+
+
+Currently the Search App is not making any requests. Instead it is using some mockup data. But
+now that we have Fetch API working, we can use that.
+
+The code that made the requests is already in place, but It is not executed because we load or
```
Comment (oscarpolanco) [inline docs/tutorial/part07.rst]: should be `requests are` instead of `requests is`

**Pull request merged**